dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/Wrapper.java

```diff
@@ -154,6 +154,12 @@ private static Wrapper makeWrapper(Class<?> c) {
         // get all public method.
         boolean hasMethod = hasMethods(methods);
         if (hasMethod) {
+            Map<String, Integer> sameNameMethodCount = new HashMap<>();

```

Comment:
it would be better if we give it a big enough initial size, such as `(int) (methods.size() / 0.75f) + 1`